src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/dotnetcli.host.json

```diff
@@ -34,25 +46,41 @@
       "shortName": "linux"
     },
     "winAppSdk": {
-      "longName": "windowsAppSdk",
+      "longName": "windows-app-sdk",
       "shortName": "win"
     },
+    "wasm": {
+      "longName": "webassembly",
+      "shortName": "wasm"
+    },
+    "wasm-pwa-manifest": {
+      "longName": "wasm-pwa-manifest",
+      "shortName": "pwa"
+    },
+    "unittest": {
+      "longName": "unittest",
+      "shortName": "unittest"
+    },
     "tests": {
-      "longName": "tests",
+      "longName": "ui-tests",
       "shortName": "tests"
     },
-    "reactive": {
```

Comment:
Most probably valid since there isn't config needed, but be aware that you remove the 'reactive' option.

Reply:
the reactive bool wasn't removed... but did change from a parameter you would specify to a computed parameter since we now use the more explicit architecture parameter with options for mvvm / mvux